sdk/core/Azure.Core/tests/public/ModelReaderWriterTests/ModelXmlOnlyTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.ClientModel;
+using Azure.Core.Tests.Common;
+using Azure.Core.Tests.Public.ModelReaderWriterTests.Models;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests.Public.ModelReaderWriterTests
+{
+    internal class ModelXmlOnlyTests : ModelTests<ModelXmlOnly>
+    {
+        protected override string WirePayload => File.ReadAllText(TestData.GetLocation("ModelXml.xml")).TrimEnd();
+
+        protected override string JsonPayload => string.Empty;
+
+        protected override string GetExpectedResult(string format)
+        {
+            var expectedSerializedString = "\uFEFF<?xml version=\"1.0\" encoding=\"utf-8\"?><Tag><Key>Color</Key><Value>Red</Value>";
+            if (format.Equals("J"))
+                expectedSerializedString += "<ReadOnlyProperty>ReadOnly</ReadOnlyProperty>";
+            expectedSerializedString += "<RenamedChildModelXml><ChildValue>ChildRed</ChildValue></RenamedChildModelXml>";
+            //TODO this is broken until we update the IXmlSerializable interface to include ModelSerializerOptions
```

Comment:
Is this comment correct?
If it represents remaining work, should there be an issue tracking it?

Reply:
Fixed